eng/scripts/Language-Settings.ps1

```diff
@@ -37,7 +37,7 @@ function Get-GoModuleVersionInfo($modPath)
 function Get-GoModuleProperties($goModPath)
 {
   $goModPath = $goModPath -replace "\\", "/"
-  if ($goModPath -match "(?<modPath>sdk/(?<serviceDir>(resourcemanager/)?([^/]+/)?(?<modName>[^/]+$)))")
+  if ($goModPath -match "(?<modPath>sdk/(?<serviceDir>((?:resourcemanager|security)/)?([^/]+/)?(?<modName>[^/]+$)))")
```

Comment:
Do we want to hardcode security here or do we want to make it more generic? 

If we do update this we also need to at least update https://github.com/Azure/azure-sdk/blob/main/eng/scripts/Query-Azure-Packages.ps1#L227 otherwise new packages will not correctly get discovered for the package index.

Reply:
https://github.com/Azure/azure-sdk-for-go/pull/19971
https://github.com/Azure/azure-sdk/pull/5534